apps/admin/frontend/src/components/signed_hash_validation_button.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
```

Comment:
Not actually a new file - just the existing `live_check_buttton.tsx` renamed and then edited via find-and-replace

Same comment elsewhere in this PR